### ISSUE TITLE
Invalid field modifiers given, allowed modifiers are IS_PUBLIC, IS_PROTECTED, IS_PRIVATE and IS_STATIC

### ISSUE THREAD

**Issue by thekabal** [phpmd/phpmd#506]
With "phpmd/phpmd" : "^2.6.0", I now receive the error:

Invalid field modifiers given, allowed modifiers are IS_PUBLIC, IS_PROTECTED, IS_PRIVATE and IS_STATIC

It does not specify which file is triggering this error, and it did not produce this error roughly a month ago. Please advise on how to debug further to help?

**Comment by thekabal** [comment]
After further examination, it appears this is due to a conflict of some kind with "squizlabs/PHP_CodeSniffer": "^3.0.2"

**Comment by ravage84** [comment]
Thanks for reporting.  Would you like to investigate further? If not, please close the issue.

By the way, you should not check your vendors anyway, as it is not your own code.

**Comment by thekabal** [comment]
Re: Vendor code, agreed! I was unable to get --exclude working, for whatever reason, when I first reported the issue. Not to worry, it was entirely user error, because by the end of the night I managed to get that working also.

No further investigation required for me, I only had two instances of the null/array, and I've corrected them both. Just wanted you wonderful folks to be aware. Keep up the great work!
